middleware/authorise.js

```diff
@@ -1,12 +1,14 @@
 module.exports = async (req, res, next) => {
+
   if (
     // allow if user is an admin
-    req.user.role !== 'admin' ||
```

Comment:
This would result to `true` for `student` user role, so the error would run even if the user was the creator of the ticket.